src/odemis/acq/align/test/spot_test.py

```diff
@@ -211,6 +211,33 @@ def test_find_grid(self):
         numpy.testing.assert_array_almost_equal(translation, numpy.array([96, 96]), decimal=3)
         numpy.testing.assert_array_almost_equal(scaling, numpy.array([12, 12]), decimal=3)
 
+    def test_find_grid_similarity(self):
+        """
+        Create an image with a grid of 8 by 8 spots. Then test if the spots are found in the correct coordinates and
+        that the rotation, scaling and translation are correct when using the similarity transformation method.
+        """
+        image = numpy.zeros((256, 256))
+        # set a grid of 8 by 8 points to 1
+        image[54:150:12, 54:150:12] = 1
+        spot_coordinates, translation, scaling, rotation, shear = spot.FindGridSpots(image, (8, 8), method='similarity')
```

Comment:
So the two methods return... exactly the same thing (excepted that shear = None)? That feels a little weak as test case. Could you find a different input where the output is really different between the 2 methods?